docs/compatibility.md

```diff
@@ -32,9 +32,15 @@ Issues of this category would impact the usage of `rbx-xml` if Roblox makes a br
 
 ### BrickColor Properties
 
-To improve readability by humans, `rbx-xml` serializes `BrickColor` properties using the element name `BrickColor`. This datatype is serialized as `int` by Roblox Studio. Since Roblox does not care about the name of the element for properties in the XML format, this has no consequences and is an easy win for readability.
+Roblox and `rbx-xml` serialize BrickColor properties using an element named `int`. When using a reflection database, this is fine because those properties get converted to BrickColor during deserialization. However, if the database used by `rbx-xml` is out of date, it won't know to do this, so any new BrickColor properties will be deserialized as integers.
```

Comment:
Since Roblox and rbx-dom are the same here, is it necessary to say anything about `int` elements or their conversion to `BrickColor`? Maybe we could just describe the interaction with the reflection database?

Reply:
After giving this some thought, I agree that mentioning the reflection database interaction should be the main point of this section. I've written a revision.